### DISCUSSION TITLE
Can we transfer data from one server to another without re-import?

### DISCUSSION THREAD

**Discussion by flydsc** [milvus-io/milvus#4343] [Q&A and General discussion] [answered]
Hi,

Thanks for this amazing tool!

I have a problem with data transferring.

I had installed milvus on one server and imported data with sqlite, then I wanted to install milvus on another server with same data. 

The question is can I just copy-paste the old milvus folders to the new server and use docker run command to mount them?

I am using sqlite right now, and can we do it on MySQL?

Hope to hear you soon.

Many thx

flydsc

**Comment by shiyu22** [comment, marked as answer]
Hi,

You are right, in your scenario, you can migrate the same version of Milvus by just copying the data in the Milvus directory, including `db` and `wal`.

And both MySQL and SQLite can be migrated in this way.
